### DISCUSSION TITLE
How to load dataset downloaded with huggingface cli?

### DISCUSSION THREAD

**Discussion by minmie** [huggingface/datasets#6818] [Q&A]
I use cli to download mozilla-foundation/common_voice_13_0(only download part of it)：

```python
huggingface-cli download --repo-type dataset --resume-download mozilla-foundation/common_voice_13_0 audio/zh-CN/train/zh-CN_train_0.tar
huggingface-cli download --repo-type dataset --resume-download mozilla-foundation/common_voice_13_0 transcript/zh-CN/train.tsv
huggingface-cli download --repo-type dataset --resume-download mozilla-foundation/common_voice_13_0 languages.py common_voice_13_0.py release_stats.py n_shards.json
```

And I load the dataset as follow:

```python
from datasets import load_dataset, DatasetDict
DATASET = 'mozilla-foundation/common_voice_13_0' 
common_voice["train"] = load_dataset(DATASET, "zh-CN", split="train[:100]")`
```

but I found it still redownload from the hub. so, how to load dataset downloaded with huggingface cli?

**Comment by wygao8** [comment]
As mentioned in issue[#6733](https://github.com/huggingface/datasets/issues/6733), this function is not implemented yet.

**Comment by XL-S224** [comment]
I also encountered this problem, when I wanted to download allenai/c4, I couldn't select the variants.Is there a solution now?
